integration_tests/ipp/container/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM zgrab2_service_base:latest
+
+RUN apt-get update
+RUN apt-get install -y cups
+RUN apt-get install -y cups-pdf
+RUN echo "ServerAlias *" >> /etc/cups/cupsd.conf
+RUN echo "FileDevice yes" >> /etc/cups/cupsd.conf
+RUN sed -i 's/Order allow,deny/Order allow,deny\'$'\n  Allow all/g' /etc/cups/cupsd.conf
+RUN sed -i 's/Listen localhost:631/Listen 0.0.0.0:631/g' /etc/cups/cupsd.conf
+RUN service cups restart
+
+# Try to make it act more container-y -- remove it from init.d and just run the daemon as the entrypoint
+#RUN service cupsd stop
```

Comment:
I think the `service cupsd stop` probably needs to be un-commented -- if the `cupsd` daemon is running / listening on the configured port, that would conflict with the foreground process from the entrypoint.

But, since this is working, something else is clearly going on (maybe the service isn't being started in the first place?) -- so maybe for now just add a TODO comment.

 - [ ] Add TODO stop service / investigate why it works without stopping the service